guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -0,0 +1,76 @@
+[id="overview-of-recommended-disaster-recovery-plans"]
+= Overview of recommended disaster recovery plans
+
+Choose a disaster recovery plan that best helps ensure the continuity of {Project} services in your deployment.
+
+//[IMPORTANT]
+//====
+//During recovery, you will need to ensure that the hostname of your {ProjectServer} does not change.
+//The IP address can change.
+//====
+
+Virtualizing your {ProjectServer}::
+How do I back up?:::
+Virtualize your {ProjectServer} and use the hypervisor tools to take virtual machine snapshots of the server.
+This method is suitable if you can run {Project} on top of virtualization.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, restore a virtual machine snapshot.
+Disadvantages and expected impact:::
+Expect some amount of data inconsistency after recovery, based on how old your last snapshot is.
+You will lose data changes that have occurred since the snapshot you are using to recover was taken.
+//While taking snapshots frequently will result in smaller amounts of data loss, creating the snapshots takes time and the snapshots themselves also take up space.
```

Comment:
```suggestion
// TODO: ???
//While taking snapshots frequently will result in smaller amounts of data loss, creating the snapshots takes time and the snapshots themselves also take up space.
```

Please fill in what still needs doing after merging.

Reply:
Removed the comment.